wrapper/src/test/java/integration/container/tests/AdvancedPerformanceTest.java

```diff
@@ -77,7 +82,7 @@ public class AdvancedPerformanceTest {
 
   private static final int TIMEOUT_SEC = 5;
   private static final int CONNECT_TIMEOUT_SEC = 5;
-  private static final int FAILOVER_TIMEOUT_MS = 300000;
+  private static final int EFM_FAILOVER_TIMEOUT_MS = 300000;
```

Comment:
This setting belongs to Failover plugin, not EFM plugin. I don't think this "EFM_" prefix makes sense here.